From 7e244e20ac6fb1b8c70be8dd0e2966a450034dda Mon Sep 17 00:00:00 2001
From: Ryan Macnak <rmacnak@google.com>
Date: Wed, 27 Nov 2024 01:25:47 +0000
Subject: [PATCH 1/2] Roll boringssl to
 cd95210465496ac2337b313cf49f607762abe286.

Bug: https://github.com/dart-lang/sdk/issues/59617
Change-Id: Ibfaa4bc4d9c4ad19e6f16b447e46cd066b376b8b
Reviewed-on: https://dart-review.googlesource.com/c/sdk/+/397681
Commit-Queue: Ryan Macnak <rmacnak@google.com>
Reviewed-by: Brian Quinlan <bquinlan@google.com>
---
 DEPS | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/DEPS b/DEPS
index afc1cf5ab05..90190242599 100644
--- a/DEPS
+++ b/DEPS
@@ -98,8 +98,8 @@ vars = {
 
   # Prefer to use hashes of binaryen that have been reviewed & rolled into g3.
   "binaryen_rev" : "459bc0797f67cb2a8fd4598bb7143b34036608d9",
-  "boringssl_gen_rev": "fef055e8d2749b82c79c8f043be1cbe5e8e4b40c",
-  "boringssl_rev": "2db0eb3f96a5756298dcd7f9319e56a98585bd10",
+  "boringssl_gen_rev": "b67a5fe651e1618baa311e452e3c188d96db2021",
+  "boringssl_rev": "cd95210465496ac2337b313cf49f607762abe286",
   "browser-compat-data_tag": "ac8cae697014da1ff7124fba33b0b4245cc6cd1b", # v1.0.22
   "cpu_features_rev": "936b9ab5515dead115606559502e3864958f7f6e",
   "devtools_rev": "711c998bb532d60c992bf48a780ab5c6595447d9",
-- 
2.47.1

